components/brave_wallet_ui/components/extension/connected-panel/index.tsx

```diff
@@ -111,6 +122,23 @@ export const ConnectedPanel = (props: Props) => {
     await copyToClipboard(selectedAccount.address)
   }, [selectedAccount.address])
 
+  const onOpenSettings = React.useCallback(() => {
+    dispatch(PanelActions.openWalletSettings())
+  }, [])
+
+  // effects
+  React.useEffect(() => {
+    if (selectedCoin === BraveWallet.CoinType.SOL) {
+      const isSolanaAccountConnected = async () => {
+        const apiProxy = getWalletPanelApiProxy()
+        await apiProxy.panelHandler.isSolanaAccountConnected(selectedAccount.address)
+          .then(result => setIsSolanaConnected(result.connected))
```

Comment:
Check if component is still mounted after promise

Reply:
Added in https://github.com/brave/brave-core/pull/13993/commits/59d1ae04bb707f4a29f6ceef4897ba0333dbd8ed